### ISSUE TITLE
IRT loss

### ISSUE THREAD

**Issue by nicolas-dufour** [yzxing87/pytorch-deep-video-prior#3]
Hi, 
I see that the loss used for IRT in the code is different than the one in the paper.
Can u pls explain why is there a difference pls?

Also, i see u divide the pixelwise difference of O_main and the net_gt by the image. What's the idea behind this?
Thank you!

**Comment by yzxing87** [comment]
Hi, our implementation is basically the same as our paper, except we use several epochs for warming up training. 

And actually, we did the division for normalization purposes before. But I think you can safely remove the division in our current implementation.
